templates/clients/docker-compose.yml

```diff
@@ -8,6 +8,9 @@ services:
   blockchain_exporter:
     container_name: blockchain_exporter
     image: nodies/blockchain_exporter
+    user: "0:0"
```

Comment:
remove hardcoded root user and change file permissions to use the docker group. add a step in setup.py that does this.